squid-blockexplorer/src/config.ts

```diff
@@ -27,7 +27,7 @@ const config: ProcessorConfig = {
         chain: process.env.CHAIN_RPC_ENDPOINT as string,
     },
     typesBundle: 'gemini-2a',
-    batchSize: 100,
+    batchSize: 20,
```

Comment:
if we do this as an environment variable it will be easier to update when "processing". After processing the first few thousand blocks I was able to bump this to 1000 with much better performance.

Reply:
good point, added